pkg/types/resource/resource_test.go

```diff
@@ -133,44 +63,28 @@ func TestNewGroupResource(t *testing.T) {
 	require.True(t, ok)
 	require.Equal(t, "v1", v1ID.Id)
 
-	roleTrait, err := GetRoleTrait(gr)
+	roleTrait, err := sdk.GetRoleTrait(gr)
```

Comment:
Probably better for another PR, but I think it makes sense to move these trait helpers to the types package as well. The `sdk` package doesn't make much sense I think, so moving more out of it feels good.